section1/exercises/booleans.rb

```diff
@@ -10,7 +10,8 @@
 p 7 > 2
 
 # YOU DO: print to the terminal the result of "hello" is equal to "Hello":
-
+p hello = "Hello"
 # YOU DO: print to the terminal the result of 3 is not equal to 4:
-
+p 3 > 4
```

Comment:
This should be `p 3 != 4`, which will have a result of true. Yours would print the value of 3 is greater than 4, which is false.